src/components/AssetsPage/index.jsx

```diff
@@ -110,34 +110,34 @@ export default class AssetsPage extends React.Component {
 
   renderAssetsPage = () => (
     <React.Fragment>
-      <div className={edxBootstrap.col}>
+      <div className="col">
         { this.renderAssetsDropZone() }
-        <div className={edxBootstrap['page-header']}>
+        <div className="page-header">
           <WrappedAssetsImagePreviewFilter />
         </div>
         { this.renderAssetsFilters() }
       </div>
-      <div className={edxBootstrap['col-10']}>
-        <div className={edxBootstrap.row}>
-          <div className={`${edxBootstrap['col-md-8']}`}>
+      <div className="col-10">
+        <div className="row">
+          <div className="col-md-8">
             <WrappedAssetsResultsCount />
           </div>
-          <div className={`${edxBootstrap['col-md-4']} ${edxBootstrap['text-right']}`}>
+          <div className={classNames('col-md-4', 'text-right')}>
```

Comment:
Are we still using classnames? Or should I just do the {[ .. ]} syntax?

Reply:
I think this is basically equivalent to `<div className="col-md-4 text-right">`. Using classNames is useful if we want to change classes based off of variables, so in this case I think a plain string is simpler.